requirements/common.txt

```diff
@@ -1,8 +1,5 @@
 alembic==0.7.6
-async==0.6.2
```

Comment:
Yep, but that async is not a Python package, but a Javascript script, which you've already added to assets/ .